site/Variables.csv

```diff
@@ -1,153 +1,153 @@
-Assay,reporterID,,,,Reporter,reporterID,string,,
-AssayHasInstrument,assayID,,,,Assay,assayID,string,,
-AssayHasInstrument,assayMethodID,,,,AssayMethod,assayMethodID,string,,
-Value,valueID,,,Primary Key,,,string,,
-Value,assayID,,,,Assay,assayID,string,,
-Value,sampleID,,,,Sample,sampleID,string,,
-Value,siteID,,,,Site,siteID,string,,
-Value,polygonID,,,,Polygon,polygonID,string,,
-Value,analysisDate,,,,,,string,,
-Value,reportDate,,,,,,string,,
-Value,measureID,,,,Measure,measureID,string,,
-Value,unitID,,,,Unit,unitID,string,,
-Value,aggregationID,,,,Aggregation,aggregationID,string,,
-Value,value,,,,,,string,,
-Value,qualityFlag,,,,,,string,,
-Value,notes,,,,,,string,,
-Aggregation,aggregationID,,,Primary Key,,,string,,
-Aggregation,descriptionEN,,,,,,string,,
-Aggregation,descriptionFR,,,,,,string,,
-AggregationScale,aggregationScaleID,,,Primary Key,,,string,,
-AggregationScale,description,,,,,,string,,
-Domain,domainID,,,Primary Key,,,string,,
-Domain,descriptionEN,,,,,,string,,
-Domain,descriptionFR,,,,,,string,,
-Measure,measureID,,,Primary Key,,,string,,
-Measure,shortName,,,,,,string,,
-Measure,fullName,,,,,,string,,
-Measure,commonNameEN,,,,,,string,,
-Measure,commonNameFR,,,,,,string,,
-Measure,descriptionEN,,,,,,string,,
-Measure,descriptionFR,,,,,,string,,
-Measure,status,,,,,,string,,
-Measure,firstReleased,,,,,,string,,
-Measure,lastUpdated,,,,,,string,,
-Measure,domainID,,,,Domain,domainID,string,,
-Measure,specimenID,,,,Specimen,specimenID,string,,
-Measure,measureGroupID,,,,MeasureGroup,measureGroupID,string,,
-Measure,measureClassID,,,,MeasureClass,measureClassID,string,,
-Measure,measurePart,,,,,,string,,
-Measure,analytePartNomenclature,,,,,,string,,
-MeasureClass,measureClassID,,,Primary Key,,,string,,
-MeasureClass,descriptionEN,,,,,,string,,
-MeasureClass,descriptionFR,,,,,,string,,
-MeasureClassHasUnit,measureClassID,,,,MeasureClass,measureClassID,string,,
-MeasureClassHasUnit,unitID,,,,Unit,unitID,string,,
-MeasureGroup,measureGroupID,,,Primary Key,,,string,,
-MeasureGroup,descriptionEN,,,,,,string,,
-MeasureGroup,descriptionFR,,,,,,string,,
-UnitScale,unitScaleID,,,Primary Key,,,string,,
-UnitScale,descriptionEN,,,,,,string,,
-UnitScale,descriptionFR,,,,,,string,,
-Specimen,specimenID,,,Primary Key,,,string,,
-Specimen,descriptionEN,,,,,,string,,
-Specimen,descriptionFR,,,,,,string,,
-Specimen,parentTable,,,,,,string,,
-Unit,unitID,,,Primary Key,,,string,,
-Unit,latexEpression,,,,,,string,,
-Unit,descriptionEN,,,,,,string,,
-Unit,descriptionFR,,,,,,string,,
-Unit,unitScaleID,,,,UnitScale,unitScaleID,string,,
-Unit,aggregationScaleID,,,,AggregationScale,aggregationScaleID,string,,
-Unit,dataType,,,,,,string,,
-UnitHasAggregation,unitID,,,,Unit,unitID,string,,
-UnitHasAggregation,aggregationID,,,,Aggregation,aggregationID,string,,
\ No newline at end of file
+tableName,variableName,variableShortFormalName,variableLabel,key,foreignKeyTable,foreignKeyVariable,variableType,minlength,maxlength,variableDesc

```

Comment:
Suggest 'uid' or 'id' for `variableShortFormalName`.

Reply:
Done!